pkg/plugins/adapters/adapters.go

```diff
@@ -42,9 +42,10 @@ func BackendUserFromSignedInUser(su *models.SignedInUser) *backend.User {
 		return nil
 	}
 	return &backend.User{
-		Login: su.Login,
-		Name:  su.Name,
-		Email: su.Email,
-		Role:  string(su.OrgRole),
+		UserID: su.UserId,
```

Comment:
Rather than just adding userId to the backend API... I think it will make sense to look at the whole token (that may have more info/groups/etc)

BUT rather than tackle that change now -- can we get the loggedInUser though context or something?  Is that a standard practice?